src/Tizen.NUI/src/public/AccessibilityManagerEventArgs.cs

```diff
@@ -0,0 +1,835 @@
+            /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public AccessibilityManager AccessibilityManager
+            {
+                get
+                {
+                    return _accessibilityManager;
+                }
+                set
+                {
+                    _accessibilityManager = value;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Event arguments that passed via ActionReadPauseResume signal
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class ActionReadPauseResumeEventArgs : EventArgs
+        {
+            private AccessibilityManager _accessibilityManager;
+
+            /// <since_tizen> 6 </since_tizen>
+            /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public AccessibilityManager AccessibilityManager
+            {
+                get
+                {
+                    return _accessibilityManager;
+                }
+                set
+                {
+                    _accessibilityManager = value;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Event arguments that passed via ActionStartStop signal
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class ActionStartStopEventArgs : EventArgs
+        {
+            private AccessibilityManager _accessibilityManager;
+
+            /// <since_tizen> 6 </since_tizen>
+            /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            public AccessibilityManager AccessibilityManager
+            {
+                get
+                {
+                    return _accessibilityManager;
+                }
+                set
+                {
+                    _accessibilityManager = value;
+                }
+            }
+        }
+
+        /*
+            // To be replaced by a new event that takes Touch
+            public class ActionScrollEventArgs : EventArgs
+            {
+              private AccessibilityManager _accessibilityManager;
+              private TouchEvent _touchEvent;
+
+              public AccessibilityManager AccessibilityManager
+              {
+                get
+                {
+                  return _accessibilityManager;
```

Comment:
this seems to be an old code pattern. in EventHandler, the first argument is object which is sender. (in this case, AccessibilityManager) so you don't need to pass AccessibilityManger instance to event subscriber through the EventArgument. This pattern is old style, and it is seen in may places. please remove this.

Reply:
Okay, Removed the old coding style and changed to the latest one. (similar to other classes)
Please  check it again.